do/loadbalancers.go

```diff
@@ -54,6 +55,19 @@ const (
 	// should use. Options are round_robin and least_connections. Defaults
 	// to round_robin
 	annDOAlgorithm = "service.beta.kubernetes.io/do-loadbalancer-algorithm"
+
+	// lbActiveTimeout is the number of seconds to wait for a load balancer to
+	// reach the active state
+	lbActiveTimeout = 60
```

Comment:
60 seconds seems very reasonable, but since these are hard coded constants I don't think it hurts to extend to 90.